app/controllers/admin/organizations_controller.rb

```diff
@@ -2,6 +2,7 @@
 class Admin::OrganizationsController < AdminController
   def edit
     @organization = Organization.find(params[:id])
+    @current_organization = Organization.find(params[:id])
```

Comment:
Do we need this assignment or could we just use the existing @organization?

Reply:
yes, you right, I forgot to remove @organizarion from this controller.